### PR TITLE
sst_importer: fix empty region after ingest lots of files

### PR DESCRIPTION
**Pull request by 3pointer**
Signed-off-by: 3pointer <luancheng@pingcap.com>

### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/br/issues/964
Problem Summary:
The origin ingests callback didn't trigger update `approximate_size` and `approximate_keys` correctly. so after BR restore or Lightning import, the TiKV cluster will display lots of empty regions.

### What is changed and how it works?

What's Changed: change ingest callback from split_check_tick to pd heartbeat tick.

### Related changes
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test

### Release note 
- Fix the issue that there are lots of empty regions after batch ingest.

### CODE REVIEW

File: components/raftstore/src/store/fsm/peer.rs [hunk 1]
```diff
@@ -3925,11 +3924,18 @@ where
...
-    fn on_ingest_sst_result(&mut self, ssts: Vec<SstMeta>) {
+    fn on_ingest_sst_result(&mut self, ssts: Vec<SSTMetaInfo>) {
+        let mut size = 0;
+        let mut keys = 0;
         for sst in &ssts {
-            self.fsm.peer.size_diff_hint += sst.get_length();
+            size += sst.total_bytes;
+            keys += sst.total_kvs;
+        }
+        self.fsm.peer.approximate_size = Some(self.fsm.peer.approximate_size.unwrap_or(0) + size);
```
Comment (Little-Wallace) [inline components/raftstore/src/store/fsm/peer.rs:3934]: Don't we need to calculate `size_diff_hint`?

Comment (3pointer) [reply components/raftstore/src/store/fsm/peer.rs:3908]: Fixed it https://github.com/tikv/tikv/pull/9924/files#diff-7a4718b9cddd27a4ef7e563024d5283cf14b4ba3abf26edc33b974b87680a5fbR3907

**Review by Little-Wallace — COMMENTED** [review]
/lgtm

**Review by Little-Wallace — APPROVED** [review]
LGTM

**Pull request merged**